src/components/Table/GridTable.tsx

```diff
@@ -847,8 +847,13 @@ export type GridColumn<R extends Kinded, S = {}> = {
     | string
     | GridCellContent
     | (DiscriminateUnion<R, "kind", K> extends { data: infer D }
-        ? (data: D, row: GridRowKind<R, K>, api: GridTableApi<R>) => ReactNode | GridCellContent
-        : (data: undefined, row: GridRowKind<R, K>, api: GridTableApi<R>) => ReactNode | GridCellContent);
+        ? (data: D, row: GridRowKind<R, K>, api: GridTableApi<R>, level: number) => ReactNode | GridCellContent
```

Comment:
Kinda wonder if we should use `opts` for the 2nd param:

```
(data: D, opts: { row, api, level }) => ..
```

So that it's easier for callers to get whatever extra opt they want...

Granted, we just did a big API change, but I guess while we're at it? :-)

Wdyt?

Reply:
Yea, I like that idea. Might was well get it done.

Reply:
And done!